packages/itwin/grouping-mapping-widget/src/components/Groups/QueryBuilder/QueryBuilder.ts

```diff
@@ -54,28 +56,43 @@ export class QueryBuilder {
   private static readonly DEFAULT_DOUBLE_PRECISION = 4;
 
   private dataProvider: PresentationPropertyDataProvider;
+  private iModelConnection: IModelConnection
   private query: Query | undefined;
 
-  constructor(provider: PresentationPropertyDataProvider) {
+  constructor(provider: PresentationPropertyDataProvider, iModelConnection: IModelConnection) {
     this.dataProvider = provider;
+    this.iModelConnection = iModelConnection
   }
 
   private isCategory(propertyField: PropertiesField): boolean {
     const classInfo = propertyField.properties[0].property.navigationPropertyInfo?.classInfo;
     return classInfo?.name === "BisCore:GeometricElement3dIsInCategory";
   }
 
+  private async getPotentialModeledElement(propertyField: PropertiesField, propertyRecord: PropertyRecord): Promise<string | undefined> {
+    const classInfo = propertyField.properties[0].property.navigationPropertyInfo?.classInfo;
+    if (propertyRecord.value?.valueFormat !== PropertyValueFormat.Primitive) return "";
```

Comment:
What's the difference between empty string and undefined?

Reply:
Incorrect. Fixed.